src/Entities/Player.h

```diff
@@ -331,6 +332,16 @@ class cPlayer :
 	void SetFoodTickTimer             (int a_FoodTickTimer);
 	void SetFoodExhaustionLevel       (double a_FoodExhaustionLevel);
 
+	// tolua_end
+
+	/** Adds to FoodLevel and FoodSaturationLevel, returns true if any food has
+	been consumed, false if player "full". Convenience method that simply calls
+	Feed(a_Food, a_Saturation) with the values from FoodInfo.
+	*/
+	bool Feed(cItemHandler::FoodInfo a_FoodInfo);
```

Comment:
I don't really see the point of adding this to `cPlayer`.  Especially if it means adding more header include dependencies.

Reply:
I can remove this if you want, I just felt that `Feed()`ing a player should be possible with a single component representation of food.

Reply:
I suppose another way to avoid having to include ItemHandler.h is to have `FoodInfo` in a namespace rather than nested in `cItemHandler`.  Then it can just be forward declared.

Reply:
I'll just remove the function instead, it's not really too important.